app/src/main/java/org/fossasia/phimpme/sharetoflickr/FlickrHelper.java

```diff
@@ -0,0 +1,67 @@
+package org.fossasia.phimpme.sharetoflickr;
+
+import com.googlecode.flickrjandroid.Flickr;
+import com.googlecode.flickrjandroid.REST;
+import com.googlecode.flickrjandroid.RequestContext;
+import com.googlecode.flickrjandroid.interestingness.InterestingnessInterface;
+import com.googlecode.flickrjandroid.oauth.OAuth;
+import com.googlecode.flickrjandroid.oauth.OAuthToken;
+import com.googlecode.flickrjandroid.photos.PhotosInterface;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+public final class FlickrHelper {
+
+	private static FlickrHelper instance = null;
+	private static final String API_KEY = "4837a3356cf697f7ef6c8b287297f19a";
```

Comment:
@the-dagger Wouldn't it be better if we remove the API keys when shifting code to master branch? As it will be easier for testing in development branch for other members of phimpme. they don't need to request api keys just to test the feature. Please suggest.

_Sent from my Xiaomi Redmi Note 4 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_